docs/user.md

```diff
@@ -348,6 +349,12 @@ spec:
 
 all files matching the pattern `*0_1*` in the bucket `s3://foo/bar` would be synced in 2 replicas and all files matching `*1_1*` would be synced in the remaining 2 replicas.
 
+## Resync
+
+For the S3 source type, the user can opt-in to resync the contents of the local directory with the source (i.e.,
+the S3 object soruce). VCK watches for any changes made to the local directory and syncs with the remote S3 object
```

Comment:
Should we also mention about the restrictions here? For example, just 1 replica is allowed, etc?

Reply:
I added the restriction on replicas there.